src/components/EditSnack/EditSnack.js

```diff
@@ -0,0 +1,36 @@
+import {connect} from 'react-redux';
+import React from 'react';
+import {setActualItem} from '../ConfirmationBox/actions';
+import ButtonList from '../ButtonList/ButtonList';
+
+class EditSnack extends React.Component {
+	render() {
+		return (
+			<ButtonList
+				items={Object.values(this.props.storeList)}
+				onClick={(id, name) => {
+					this.props.setActualItem(name);
+				}}
+			/>
+		);
+	}
+}
+
+const mapStateToProps = state => {
+	return {
```

Comment:
We're defining components and containers in separate files elsewhere, should we separate these to be consistent?